README.md

```diff
@@ -51,10 +51,10 @@ Generally, your project will have better chances of being accepted if:
 - You can demonstrate that the project will be **maintained** after completion of the grant, if applicable.
 - Your team has **proven experience** with relevant languages, technologies, and/or a strong technical background. GitHub profiles of team members should be included in your application.
 - Your application is **rich in technical details** and well-defined.
-- Your project uses the RISC-V compatible version of the ink! language, [v6.X releases](https://use.ink/6.x/). The current cohort is focused on funding developments willing to push the ecosystem forward by leveraging ink!'s stack in its most ecosystem-aligned version.
 
 Additionally, all projects must fulfill the following requirements:
 
+- Your project uses the RISC-V compatible version of the ink! language, [v6.X releases](https://use.ink/6.x/). The current cohort is focused on funding developments willing to push the ecosystem forward by leveraging ink!'s stack in its most ecosystem-aligned version.
```

Comment:
That page says 

> We plan to release ink! 6.0.0-alpha in February and will update this section with testnet information then.

Might be time for an update. @cmichi?

Reply:
@cmichi I've created an ink-docs PR to update release target to March https://github.com/use-ink/ink-docs/pull/360